symplyphysics/definitions/impulse_due_to_force.py

```diff
@@ -0,0 +1,54 @@
+from sympy import Eq, Integral
+from symplyphysics import (
+    units,
+    Quantity,
+    Function,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The impulse due to force exerted on body during collision is the measure of both the
```

Comment:
Use `momentum` instead of `impulse` everywhere.

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/8e9bd887-3b81-49c9-8c57-82033a81bc80)
This very quantity is called impulse though